sdk/js/src/utils/consts.ts

```diff
@@ -339,7 +339,7 @@ const MAINNET = {
     nft_bridge: undefined,
   },
   seievm: {
-    core: undefined,
+    core: "0x07782FCe991dAb4DE7a3124032E534A0D059B4d8",
```

Comment:
This explorer also seems like it's supposed to only work for arctic-1 (public devnet) where it shows this contract  https://seitrace.com/address/0x07782FCe991dAb4DE7a3124032E534A0D059B4d8

Reply:
Moved address to TESTNET.